docs/guides/main.rst

```diff
@@ -22,9 +22,7 @@ Beets works on `Python 2.7`_ and Python 3.4 or later.
   official package (`Debian details`_, `Ubuntu details`_), so try typing:
   ``apt-get install beets``. But the version in the repositories might lag
   behind, so make sure you read the right version of these docs. If you want
-  the latest version, you can get everything you need to install with pip
-  as described below by running:
-  ``apt-get install python-dev python-pip``
+  the latest version, you can use the `snap <snap_>`_.
```

Comment:
pip still works... maybe worth noting both options

Reply:
Can I reply with more of a meta question? pip install works for all platforms, so should it not be more of a generic installation method?